### PR TITLE
Update Cargo.toml

### PR DESCRIPTION
**Pull request by goshawk-3**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -2,8 +2,16 @@
...
+repository = "https://github.com/dusk-network/consensus"
+description = "An implementation of Succinct Attestation consensus protocol"
+categories = ["cryptography", "cryptography::cryptocurrencies"]
+keywords = ["consensus", "dusk", "cryptocurrency", "blockchain"]
+license = "MPL-2.0"
+exclude = [".github/*", ".gitignore", ".env", ".vscode"]
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[lib]
+name = "consensus"
```
Comment (herr-seppia) [inline Cargo.toml:13]: This crate name has been already taken on crates.io

**Review by herr-seppia — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (goshawk-3) [reply Cargo.toml:13]: Good point! Any suggestions? E.g `dusk-consensus` ?

File: Cargo.toml [hunk 2]
```diff
@@ -2,8 +2,16 @@
 name = "consensus"
 version = "0.1.0"
 edition = "2021"
+repository = "https://github.com/dusk-network/consensus"
+description = "An implementation of Succinct Attestation consensus protocol"
+categories = ["cryptography", "cryptography::cryptocurrencies"]
+keywords = ["consensus", "dusk", "cryptocurrency", "blockchain"]
+license = "MPL-2.0"
+exclude = [".github/*", ".gitignore", ".env", ".vscode"]
```
Comment (herr-seppia) [inline Cargo.toml:10]: I believe these are not required

> If include is not specified, then the following files will be excluded:
> 
> * If the package is not in a git repository, all "hidden" files starting with a dot will be skipped.
> * If the package is in a git repository, any files that are ignored by the [gitignore](https://git-scm.com/docs/gitignore) rules of the repository and global git configuration will be skipped.

_src here: https://doc.rust-lang.org/cargo/reference/manifest.html#the-exclude-and-include-fields_

**Review by herr-seppia — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (herr-seppia) [reply Cargo.toml:13]: Yes, I'm for that

File: Cargo.toml (hunk 2 above)
Comment (goshawk-3) [reply Cargo.toml:10]: `cargo publish --dry-run` fails if `exclude` does not list these files.

Comment (herr-seppia) [reply Cargo.toml:10]: Oh I see, that's because is a git repo, so the .gitignore rules

**Review by herr-seppia — APPROVED** [review]
LGTM

**Pull request merged**